modules/core/core-git/src/main/java/org/eclipse/dirigible/core/git/command/PullCommand.java

```diff
@@ -27,6 +27,7 @@
 import org.eclipse.dirigible.core.publisher.service.PublisherCoreService;
 import org.eclipse.dirigible.core.workspace.api.IProject;
 import org.eclipse.dirigible.core.workspace.api.IWorkspace;
+import org.eclipse.dirigible.repository.api.IRepositoryStructure;
```

Comment:
Is this import needed?

Reply:
No. It must have remained after some tests. 
Will be more careful next time.